pkg/kubectl/cmd/util/helpers.go

```diff
@@ -45,7 +46,8 @@ import (
 )
 
 const (
-	ApplyAnnotationsFlag = "save-config"
+	ApplyAnnotationsFlag      = "save-config"
+	ChangeSourceAnnotationKey = "kubectl.kubernetes.io/change-source"
```

Comment:
I'd like the same annotation to mostly make sense even for mutations that don't originate from kubectl, so I'd prefer not to have "command" in the key name. Perhaps that suggests "kubectl" shouldn't be in the prefix, either.

kubernetes.io/change-cause ?